_posts/events/2015-07-01-mumttt.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+categories: [event]
+event_name: "The 2nd Workshop on Multi-word Units in Machine Translation and Translation Technology (MUMTTT 2015)"
+event_url: "http://www.tradulex.com/varia/MUMTTT2015.pdf"
+event_location: "Malaga, Spain"
+event_submission_date: "2015-03-31"
+host_name: "europhras 2015"
+host_url: ""
+siglex_endorsed: yes
+---
+The 2nd Workshop on Multi-word Units in Machine Translation and Translation Technology (MUMTTT 2015)
+
+Malaga, Spain, 1-2 July 2015
+
+Endorsed by:
+
+SIGLEX, the Special Interest Group on the Lexicon of the ACL
+PARSEME, the European IC1207 COST Action
+
+BAKGROUND 
+
+Following the successful workshop on Multi-word Units in Machine Translation and Translation Technology held in conjunction with the MT SUMMIT 2013 conference in Nice (MUMTTT 2013, 3 September 2013), the editors invite submissions for a volume on the same topics to be published by John Benjamins during the second half of 2015.
+This volume will feature contributions reporting recent advances in the topics which cut across Linguistics, Computational Linguistics, Translation Studies and Computational Phraseology. We invite submissions describing substantial work related to the processing of multi word units (MWUs) on topics including, but not limited to, the ones listed below. Authors of the MUMTTT 2013 workshop papers are invited to submit extended and revised versions of their papers, but this call is open to all researchers working on relevant topics and willing to share their work. Submissions will be reviewed by at least three reviewers.
+
+TOPICS (the list below is indicative but not exhaustive):
+ * Lexical, syntactic, semantic and translational aspects in MWU representation
+ * Collostructional analysis and MWU
+ * Automatic extraction of multilingual MWU resources
+ * Identification and acquisition of multi-word terms and their variants
+ * Development and use of handcrafted MWU linguistic resources in MT
+ * MWUs in Computer-assisted Translation
+ * MWUs in Machine Translation
+ * Paraphrasing of MWUs applied to the improving of MT
+ * MWUs and word alignment techniques
+ * Learning semantic information about MWUs from monolingual, parallel or comparable corpora
+ * Multilingualism and MWU processing
+ * MWU-centred MT evaluation
+ * Compilation of resources for extraction and translation of multiword units
+ * Corpora annotated with MWUs for translation studies
+
+SUBMISSIONS 
+
+We invite expressions of interest by 1 November 2014 by email to both contact editors below, including the chapter title, list of prospective authors and affiliation, as well as a 250-word abstract of your contribution.
+The full submission is expected to be approximatively 5,000 words in length, excluding citations and appendices, and should be formatted according to the publisher’s author guidelines (https://benjamins.com/jbp/additional/jb-guidelines-manuscript-submission-apa.pdf). The deadline for the final contributions is 31 March 2015.
```

Comment:
Put `<` and `>` around the URL so that it is rendered as a link.

Reply:
Done